core/movement/motor/motor.cc

```diff
@@ -0,0 +1,40 @@
+#include "motor.hh"
+#include <math.h>
+
+Motor::Motor(const int &enable_pin, const int &forward_pin, const int &backward_pin)
+{
+    m_pins.pwm = enable_pin;
+    m_pins.forward = forward_pin;
+    m_pins.backward = backward_pin;
+    m_rpm = 0;
+    m_pwm = 0;
+}
+
+void Motor::set_throttle(int pwm_value, MotorDirection dir)
+{
+    auto bounded_pwm = fmin(255, fmax(0, pwm_value));
+    m_pwm = bounded_pwm;
+    m_dir = dir;
+
+    switch (m_dir)
+    {
+    case STOP:
+    {
+        m_pwm = 0;
+        digitalWrite(m_pins.backward, LOW);
+        digitalWrite(m_pins.forward, LOW);
+    }
+    case FORWARD:
```

Comment:
Yep, good point. This will be implemented in a higher level layer that controls both motors within a single instance. I can open an [issue](https://github.com/180D-FW-2020/Team3/issues/4) for this.